policyengine_us/parameters/gov/states/ca/cdss/child_care/rate_ceilings/standard.yaml

```diff
@@ -0,0 +1,366 @@
+description: California CalWORKs Child Care provides up to this child care reimbursement based on specific conditions.
+
+metadata:
+  label: California CalWORKs Child Care Standard Regional Market Rate Ceilings
+  type: single_amount
+  threshold_unit: year
+  amount_unit: currency-USD
+  breakdowns:
+    - [child_care_centers, family_child_care_homes, license_exempt]
+    - [hourly, daily, weekly, monthly]
+    - [full_time, part_time]
+  reference:
+    - title: Los Angeles Department of Public Social Services E-Policy | RMR Ceilings Exhibit I
+      href: http://epolicy.dpss.lacounty.gov/epolicy/epolicy/server/general/projects_responsive/ePolicyMaster/index.htm?&area=general&type=responsivehelp&ctxid=&project=ePolicyMaster#t=mergedProjects%2FChild%20Care%2FChild_Care%2F1210_8_Regional_Market_Rate_Ceilings%2F1210_8_Regional_Market_Rate_Ceilings.htm%23Contactbc-13&rhtocid=_3_3_8_12
+
+child_care_centers:
+  hourly:
+    full_time:
+      brackets:
+        - amount:
+            2018-01-01: 0
+          threshold:
+            2018-01-01: 0
+        - amount:
+            2018-01-01: 0
+          threshold:
+            2018-01-01: 2
+        - amount:
+            2018-01-01: 0
+          threshold:
+            2018-01-01: 6
+    part_time:
+      brackets:
+        - amount:
+            2018-01-01: 16.28
+          threshold:
+            2018-01-01: 0
+        - amount:
+            2018-01-01: 12.19
+          threshold:
+            2018-01-01: 2
+        - amount:
+            2018-01-01: 11.24
+          threshold:
+            2018-01-01: 6
+  daily:
+    full_time:
+      brackets:
+        - amount:
+            2018-01-01: 96.53
+          threshold:
+            2018-01-01: 0
+        - amount:
+            2018-01-01: 67.37
+          threshold:
+            2018-01-01: 2
+        - amount:
+            2018-01-01: 55.72
+          threshold:
+            2018-01-01: 6
+    part_time:
+      brackets:
+        - amount:
+            2018-01-01: 0
+          threshold:
+            2018-01-01: 0
+        - amount:
+            2018-01-01: 0
+          threshold:
+            2018-01-01: 2
+        - amount:
+            2018-01-01: 0
+          threshold:
+            2018-01-01: 6
+  weekly:
+    full_time:
+      brackets:
+        - amount:
+            2018-01-01: 398.15
+          threshold:
+            2018-01-01: 0
+        - amount:
+            2018-01-01: 301.55
+          threshold:
+            2018-01-01: 2
+        - amount:
+            2018-01-01: 234.53
+          threshold:
+            2018-01-01: 6
+    part_time:
```

Comment:
in the full or part time variable calculation include where statements the specify for certain time categories, only full or part time is possible